composer.json

```diff
@@ -13,13 +13,15 @@
   ],
   "require": {
     "php": ">=7.4",
+    "composer-runtime-api": "^2.2.2",
     "symfony/process": "~2.5|~3.0|~4.4|^6",
     "symfony/dependency-injection": "~2.6|~3.0|~4.4|^6",
     "drupal/core-utility": "^8.4 || ^9 || ^10@beta"
   },
   "require-dev": {
     "composer/installers": "^2.1",
     "drupal/coder": "~8.3.0",
+    "drush/drush": "^11 || ^12",
```

Comment:
Since only Drush >=12  is supported, Drush 11 should not be on the list.

https://www.drush.org/12.4.4/install/#drupal-compatibility